diagnostic.md

```diff
@@ -6,70 +6,74 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+resources :books only: [:show]
+
+# OR
+
+get '/books/:id' => 'books#show' # This is my best guess for the second method, based on how the default user actions are defined
 ```
```

Comment:
We were actually just looking for these two:

```
get '/books/:id', to: 'books#show'
get '/books/:id' => 'books#show'
``` 
I can count that, it shows a strong understanding of rails routing.